docs/release-notes.md

```diff
@@ -26,6 +26,7 @@
 * [1224912] (https://bugzilla.redhat.com/show_bug.cgi?id=1224912) - Filter "Last modified by translators other than <user>"
 * [1213630](https://bugzilla.redhat.com/show_bug.cgi?id=1213630) - Webhook header needs to include cryptographic signature in header for identification
 * [1214502](https://bugzilla.redhat.com/show_bug.cgi?id=1214502) -  RFE: Grant project creation permission to certain sets of users
+* [ZNTA-108](https://zanata.atlassian.net/browse/ZNTA-108) - Glossary management
```

Comment:
All this work deserves more than 2 words :)

Maybe `Improved glossary management: add, edit and delete individual glossary entries`